controllers/hetznerbaremetalmachine_controller_test.go

```diff
@@ -825,4 +833,239 @@ var _ = Describe("HetznerBareMetalMachineReconciler", func() {
 			})
 		})
 	})
+
+	Context("Test EnqueueRequestsFromMapFunc BareMetalHostToBareMetalMachines", func() {
+		It("should enqueue the second HetznerBareMetalMachine, when the first gets deleted", func() {
+			// We test this part in BareMetalHostToBareMetalMachines:
+			// We have a free host. Trigger a matching HetznerBareMetalMachine to be reconciled.
+			// To ensure this test works, uncomment the lines below
```

Comment:
what do you mean by this uncommenting?

Reply:
I remove the lower part from the comment. It does not matter.